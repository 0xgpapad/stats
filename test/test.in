100 mon
150 mon
this -100 day
this 200 day
this300 day
100 200
200 300
Ending in dot 100.
Ending in dot 200.
Startint 100
Startint 200.2
Startint 300
Startfloat 100.1
Startfloat 200
Startfloat 300.1
Floating 100 1.5
Floating 200 2.5
Same number 100 equals
Same number 100 equals
