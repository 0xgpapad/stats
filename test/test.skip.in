Small number 1, Large number 1000
Small number 1, Large number 2000
Small number 2, Large number 5000
Small number 2, Large number 100000
