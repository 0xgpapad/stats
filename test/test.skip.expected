Small number 1, Large number 1000-2000(1500+/-5e+02)
Small number 2, Large number 5000-100000(52500+/-4.8e+04)
